libcontainer/factory_nabla.go

```diff
@@ -87,14 +87,17 @@ func applyPauseHack(config *configs.Config, containerRoot string) (*configs.Conf
 func createRootfsISO(config *configs.Config, containerRoot string) (string, error) {
 	rootfsPath := config.Rootfs
 	targetISOPath := filepath.Join(containerRoot, "rootfs.iso")
+	if err := os.MkdirAll(filepath.Join(rootfsPath+"/etc"), 0755); err != nil {
```

Comment:
`filepath.Join` takes in multi args, so usage would be `filepath.Join(rootfsPath, "/etc/")`